code/game/objects/items/weapons/twohanded.dm

```diff
@@ -373,7 +373,8 @@
 		if(!isliving(target))
 			return
 		else
-			target.Weaken(4)
+			if(!istype(src, /obj/item/weapon/twohanded/chainsaw/rubber))

```

Comment:
well you could do some magic if you want, it would probably be nicer than this
hit the f1 menu in dream maker and lookup the syntax for `call`, i believe you can directly call procs on certain paths, then do `call(something i don't remember, attack()(put all the arguments in this set of brackets)`